evm_loader/cli/src/logs.rs

```diff
@@ -1,18 +1,36 @@
-use serde::{ Deserialize, Serialize };
+use serde::{ Deserialize };
 use fern::{ Dispatch };
 
-#[derive(Deserialize,Serialize)]
+#[derive(Deserialize)]
 #[derive(Default)]
 pub struct LogContext {
     req_id: String,
 }
 
+impl LogContext {
+    pub fn new(id: String) -> LogContext {
+        LogContext {
+            req_id: id,
+        }
+    }
+}
 
-const LOG_MODULES: [&str; 2] = [
+
+const LOG_MODULES: [&str; 11] = [
   "neon_cli",
   "neon_cli::account_storage",
+  "neon_cli::commands::cancel_trx",
+  "neon_cli::commands::create_ether_account",
+  "neon_cli::commands::create_program_address",
+  "neon_cli::commands::deploy",
+  "neon_cli::commands::emulate",
+  "neon_cli::commands::get_ether_account_data",
```

Comment:
neon_cli::commands::get_ether_account_data - 43 symbols
Will it fit in the appropriate box?

Reply:
You're right, it won't fit. Is it Ok, if it won't fit?

Reply:
I've checked, the field will be extended up to 43 symbols. I think, It's Ok.